lightning-rapid-gossip-sync/src/processing.rs

```diff
@@ -236,14 +237,16 @@ mod tests {
 
 	use lightning::ln::msgs::DecodeError;
 	use lightning::routing::gossip::NetworkGraph;
+	use lightning::util::test_utils::TestLogger;
 
 	use crate::error::GraphSyncError;
 	use crate::RapidGossipSync;
 
 	#[test]
 	fn network_graph_fails_to_update_from_clipped_input() {
 		let block_hash = genesis_block(Network::Bitcoin).block_hash();
```

Comment:
the pre-test parametrization is getting to a point where perhaps creating a helper method, such as get_test_network_graph(), would be helpful